projects/observability/src/public-api.ts

```diff
@@ -184,9 +184,9 @@ export * from './shared/components/timeline-card-list/timeline-card-list.module'
 export * from './shared/components/interval-select/interval-select.component';
 export * from './shared/components/interval-select/interval-select.module';
 
-// Top N Chart
-export * from './shared/components/top-n/top-n-chart.module';
-export * from './shared/components/top-n/top-n-chart.component';
+// Sort N Chart
+export * from './shared/components/sort-n/sort-n-chart.module';
```

Comment:
To save some effort, we should probably discuss and agree on names before you go through the effort of implementing the change - I didn't like top N and I'm not sure I understand the change to sort-n.

Reply:
yeah. learnt it the hard way